.github/workflows/release.yml

```diff
@@ -97,6 +96,30 @@ jobs:
           name: kwctl-linux-${{ matrix.targetarch }}
           path: kwctl-linux-${{ matrix.targetarch }}.zip
 
+      - name: Generate SBOM
+        run: |
+          make download-spdx-sbom-generator sbom
```

Comment:
It's a matter of personal taste, but what about using https://github.com/kubewarden/github-actions/tree/v1/sbom-generator-installer to get the sbom tool? Just to be consistent with what we are doing with our policies

Reply:
True. Let me fix it.